frontend/src/component/user/Auth0Auth.tsx

```diff
@@ -0,0 +1,224 @@
+                setErrors({
+                    apiError: 'Invalid password and username combination.',
+                });
+            } else {
+                setErrors({
+                    apiError: 'Unknown error while trying to authenticate.',
+                });
+            }
+        }
+    };
+
+    const handleSubmit: FormEventHandler<HTMLFormElement> = async evt => {
+        evt.preventDefault();
+
+        if (!username) {
+            setErrors(prev => ({
+                ...prev,
+                usernameError: 'This is a required field',
+            }));
+        }
+        if (!password) {
+            setErrors(prev => ({
+                ...prev,
+                passwordError: 'This is a required field',
+            }));
+        }
+
+        if (!password || !username) {
+            return;
+        }
+
+        try {
+            await passwordAuth(authDetails.path, username, password);
+            refetchUser();
+            navigate(redirect, { replace: true });
+        } catch (error: any) {
+            if (
+                error instanceof NotFoundError ||
+                error instanceof BadRequestError
+            ) {
+                setErrors(prev => ({
+                    ...prev,
+                    apiError: 'Invalid login details',
+                }));
+                setPassword('');
+                setUsername('');
+            } else if (error instanceof AuthenticationError) {
+                setErrors({
+                    apiError: 'Invalid password and username combination.',
+                });
+            } else {
+                setErrors({
+                    apiError: 'Unknown error while trying to authenticate.',
+                });
+            }
+        }
+    };
+
+    const renderLoginForm = () => {
+        const { usernameError, passwordError, apiError } = errors;
+
+        return (
+            <ConditionallyRender
+                condition={!authDetails.defaultHidden}
+                show={
+                    <form onSubmit={handleSubmit}>
+                        <ConditionallyRender
+                            condition={Boolean(apiError)}
+                            show={
+                                <StyledAlert severity="error">
+                                    {apiError}
+                                </StyledAlert>
+                            }
+                        />
+
+                        <StyledDiv>
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                type="button"
+                                style={{ width: '100%', margin: '1rem auto' }}
+                                data-testid={LOGIN_BUTTON}
+                                onClick={handleAuth0}
+                            >
+                                Sign in with Microsoft Enterprise
```

Comment:
Has this project internationalization support ?

Reply:
No, it does not support